app/src/test/java/com/fstyle/structure_android/data/source/UserRepositoryTest.java

```diff
@@ -28,23 +28,29 @@
     private static final String USER_LOGIN_1 = "abc";
     private static final String USER_LOGIN_2 = "def";
 
+    @InjectMocks
+    private UserRepositoryImpl mUserRepository;
     @Mock
-    NameApi mNameApi;
-
-    private UserRepository mUserRepository;
+    UserLocalDataSource mLocalDataSource;
+    @Mock
+    UserRemoteDataSource mRemoteDataSource;
 
     @Before
     public void setUp() throws Exception {
         MockitoAnnotations.initMocks(this);
-        mUserRepository = new UserRepositoryImpl(null, new UserRemoteDataSource(mNameApi));
     }
 
     @Test
-    public void searchUsers200ResponseCodeInvokesCorrectApiCalls() {
+    public void searchUsers_200ResponseCode_InvokesCorrectApiCalls() {
```

Comment:
ERROR: [MethodNamingConventions] Method names should not contain underscores
https://pmd.github.io/pmd-5.5.1/pmd-java/rules/java/naming.html#MethodNamingConventions